### PR TITLE
Document a new SME support routine to query the current value of VG.

### PR DESCRIPTION
**Pull request by kmclaughlin-arm**

### CODE REVIEW

**Review by rsandifo-arm — COMMENTED** [review]
LGTM, but I think it'd be worth mentioning this change in the change history table.

File: aapcs64/aapcs64.rst [hunk 1]
```diff
@@ -2269,6 +2273,38 @@ a lazy save, with the subroutine having the following properties:
 * The only memory modified by the subroutine (if any) is stack memory
   below the incoming SP.
 
+``__arm_get_current_vg``
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+**(Beta)**
+
+Platforms that support SME must provide a subroutine to query the current
+value of VG, with the subroutine having the following properties:
```
Comment (sallyarmneale) [inline aapcs64/aapcs64.rst:2282]: change to:

and the subroutine must have the following properties:

**Review by sallyarmneale — COMMENTED** [review]
No changes required

File: aapcs64/aapcs64.rst (hunk 1 above)
Comment (stuij) [reply aapcs64/aapcs64.rst:2282]: addressed by https://github.com/ARM-software/abi-aa/pull/278

**Pull request merged**